scripts/manila-share-sync.py

```diff
@@ -146,6 +146,8 @@ def sync_share_size(self, shares, dry_run=True):
                     continue
 
             if size != vsize:
+                if share['volume']['snap_percent'] == 50 and size == vsize/2:
```

Comment:
Please no hardcode. [Manila nanny is mounting manila.conf file](https://github.com/sapcc/helm-charts/blob/master/openstack/nannies/templates/manila-nanny-deployment.yaml#L30-L34) and thus this can be read by Nanny to find out the snapreserve_percentage as well as if the new setting / new style volumes are enabled.

Reply:
updated to read from config file, with default value of 50